src/main.py

```diff
@@ -1,64 +1,16 @@
-from typing import Any
+from pathlib import Path
 
-import pandas as pd
-import sqlalchemy
-from dune_client.client import DuneClient
-from dune_client.query import QueryBase
-from pandas import DataFrame
-from sqlalchemy import create_engine
-
-from src.config import Env, RuntimeConfig
-from src.mappings import DUNE_TO_PG
-
-DataTypes = dict[str, Any]
-
-
-def reformat_varbinary_columns(
-    df: DataFrame, varbinary_columns: list[str]
-) -> DataFrame:
```

Comment:
We lost all our types.

Reply:
This is one major downside of the approach, I agree. Since functions have to reach into the `bag` and pull arbitrary properties contained in it, it's not immediately obvious what exactly `reformat_varbinary_columns` would need in order to operate. I'll try to think of a way to fix this.